src/styles/annotator/notebook.scss

```diff
@@ -0,0 +1,21 @@
+@use '../variables' as var;
+@use '../mixins/molecules';
+
+.notebook-outer {
+  // TODO: CSS namespace conflicts?
+  box-sizing: border-box;
+  position: absolute;
+  top: 0;
+  left: 0;
+  width: 100vw;
```

Comment:
I haven't taken the time to see if we're OK using `v*` units yet. Will take more care as we progress.

Reply:
Using `vw` units is fine from a [browser support](https://caniuse.com/viewport-units) standpoint, and I can't think of any other reasons to avoid them.